### PR TITLE
add html, audio types, fix video implementation

### PR DESCRIPTION
**Pull request by trust0212**

### CODE REVIEW

**Review by chinu-dev — APPROVED** [review]
I don’t see audio or html type used. But we can do it later

**Comment by trust0212** [comment]
Actually Audio and Html are kind of Video.
But we need to split them for frontend implementation.

File: src/eth/helpers.ts [hunk 1]
```diff
@@ -319,8 +294,51 @@ export const nftportAssetToCollectible = async (
       }
 
       videoUrl = [animation_url, animation_original_url, ...imageUrls].find(
-        url => url && SUPPORTED_VIDEO_EXTENSIONS.some(ext => url.endsWith(ext)),
```
Comment (sacred-danny) [inline src/eth/helpers.ts:322]: SUPPORTED_VIDEO_EXTENSIONS does not have html and htm extension right no
So if the url ends with html or html, videoUrl can be null
You should add html and htm to OPENSEA_VIDEO_EXTENSIONS

**Review by sacred-danny — CHANGES_REQUESTED** [review]

**Comment by trust0212** [comment]
[image: image.png]
we already have htm, html extension in SUPPORTED_VIDEO_EXTENSIONS.
Is it not enough?

File: src/eth/helpers.ts [hunk 2]
```diff
@@ -15,9 +15,11 @@ const OPENSEA_VIDEO_EXTENSIONS = [
   'ogv',
   'ogg',
   'mov',
+  'html',
+  'html'
```
Comment (sacred-danny) [inline src/eth/helpers.ts:19]: you need to change html to htm

**Review by sacred-danny — APPROVED** [review]

**Pull request merged**